services/autoscaling/src/simcore_service_autoscaling/models.py

```diff
@@ -10,3 +14,28 @@ class EC2Instance(BaseModel):
     name: str
     cpus: PositiveInt
     ram: ByteSize
+
+
+class SimcoreServiceDockerLabelKeys(BaseModel):
+    # NOTE: in a next PR, this should be moved to packages models-library and used
```

Comment:
i wonder how many variants we have of this :)

Reply:
quite a few... quite  a few..